core/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -213,6 +214,22 @@ class BasicWorkflow extends EnsimeSpec
           expectMsgPF() {
             case RefactorResult(1234, RefactorType.Rename, List(`fooFile`), _) =>
           }
+
+          import scala.reflect.runtime.universe._
+          import scala.reflect.runtime.{ currentMirror => cm }
+          import scala.tools.reflect.ToolBox
+
+          val tree1 = q"List(1, 2, 3).head + 2"
+          val ast1 = showRaw(cm.mkToolBox().typecheck(tree1), printTypes = true)
```

Comment:
I'd prefer to hardcode the expected response... it's more reliable than finding it out through the toolbox. We can always have multiple valid expectations to handle changes in the scala version.